tests/framework/console/ControllerTest.php

```diff
@@ -132,4 +132,18 @@ public function testHelpOptionWithModule()
         $this->assertFalse(FakeController::getWasActionIndexCalled());
         $this->assertEquals(FakeHelpController::getActionIndexLastCallParams(), ['news/posts/index']);
     }
+
+
+    /**
+     * Tests if action help does not include (class) type hinted arguments.
+     * @see #10372
+     */
+    public function testHelp()
```

Comment:
Could be named `testHelpSkipsTypehintedArguments()`. Long but name itself explains what it does.